### PR TITLE
Cleanup unused messages

### PR DESCRIPTION
**Pull request by fvantienen**
This is removing unused messages as discussed in #27. I've only done it up to 100, but wanted to know if this look ok, or should we remove even more messages/cleanup old code?

### CODE REVIEW

**Comment by flixr** [comment]
Looks like you removed some messages that are actually removed...
I think it would first be better to make a list of which messages to remove in #27

File: message_definitions/v1.0/messages.xml [hunk 1]
```diff
@@ -266,12 +260,7 @@
     <field name="value" type="float"/>
   </message>
 
-
-  <message name="MARK" id="32">
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: This one is used by the GCS to leave a mark on the map (the button 'mark' on the strip that probably nobody knows...)

**Comment by dewagter** [comment]
great idea, but quite an invasive change. maybe something to apply only just after the 5.10 release ?

**Comment by dewagter** [comment]
What about a script to parse all telemetry.xml and all *.c/*.h/*.ml files for messages actually being used?

**Comment by gautierhattenberger** [comment]
That would be a start, can you do it ?

**Comment by podhrmic** [comment]
See #36 for some deleted messages

**Comment by podhrmic** [comment]
Closing in favour of #41

**Pull request closed without merging**